tsconfig.json

```diff
@@ -3,23 +3,24 @@
 	"compilerOptions": {
 		"alwaysStrict": true,
 		"lib": ["esnext"],
+		"strict": true,
 		"module": "commonjs",
 		"noUnusedParameters": true,
 		"outDir": "./dist",
 		"sourceMap": true,
-		"types": ["node"],
-        "declaration": true,
-        "noUnusedLocals": true,
-        "removeComments": false,
-        "target": "esnext"
+		"types": ["node", "discord.js", "klasa"],
+		"declaration": true,
+		"declarationMap": true,
+		"noUnusedLocals": true,
+		"removeComments": false,
+		"target": "esnext"
 	},
 	"exclude": [
-		"node_modules",
 		"bin",
 		"examples",
 		"scripts"
```

Comment:
Exclude is not needed. Exclude is meant to exclude files previously included in `include.` From [the docs](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#details)

> If the "files" and "include" are both left unspecified, the compiler defaults to including all TypeScript (.ts, .d.ts and .tsx) files in the containing directory and subdirectories except those excluded using the "exclude" property. JS files (.js and .jsx) are also included if allowJs is set to true. If the "files" or "include" properties are specified, the compiler will instead include the union of the files included by those two properties. Files in the directory specified using the "outDir" compiler option are excluded as long as "exclude" property is not specified.